lib/internal/heap_utils.js

```diff
@@ -1,16 +1,30 @@
 'use strict';
 const {
-  Symbol
+  Symbol,
+  Uint8Array,
 } = primordials;
 const {
   kUpdateTimer,
   onStreamRead,
 } = require('internal/stream_base_commons');
 const { owner_symbol } = require('internal/async_hooks').symbols;
 const { Readable } = require('stream');
+const { validateObject, validateBoolean } = require('internal/validators');
+const { kEmptyObject } = require('internal/util');
 
 const kHandle = Symbol('kHandle');
 
+function getHeapSnapshotOptions(options = kEmptyObject) {
+  validateObject(options, 'options');
+  const {
+    exposeInternals = false,
+    exposeNumericValues = false,
+  } = options;
```

Comment:
Updated to use destructuring + default values because assignment doesn't work with kEmtpyObject. @jasnell @aduh95